examples/prepy310/vector_index.py

```diff
@@ -107,7 +107,7 @@ def search(client: PreviewVectorIndexClient, index_name: str) -> None:
 def delete_items(client: PreviewVectorIndexClient, index_name: str) -> None:
     item_ids_to_delete = ["test_item_1", "test_item_3"]
     _logger.info(f"Deleting items: {item_ids_to_delete}")
-    delete_response = client.delete_item_batch(index_name, ids=item_ids_to_delete)
+    delete_response = client.delete_item_batch(index_name, filter=item_ids_to_delete)
```

Comment:
just wondering, why `filter` and not `filters`? Looks like this is an array

Reply:
In a coming PR this will accept a (single) filter expression or (for convenience) a list of ids to select.